core/src/main/java/com/graphhopper/routing/profiles/HazmatWater.java

```diff
@@ -0,0 +1,22 @@
+package com.graphhopper.routing.profiles;
+
+/**
+ * Defines general restrictions for the transport of goods through water protection areas.<br>
+ * If not tagged it will be {@link #YES}
+ */
+public enum HazmatWater {
+    YES("yes"), NO("no");
```

Comment:
Should we introduce "permissive"? As this is used over 2000 times (comparable order like "no" that has ~3500 ways)

Reply:
"permissive" means that the drive *should* use a other route. Is our routing able to work with such soft restrictions?

Reply:
Sure, this is possible.